### PR TITLE
add assertions to proc_single

### PR DESCRIPTION
**Pull request by leoscholl**
so that we can better detect errors in pre-processing. i also added downsampling to the eyetracking data since it's wasting a lot of space at 25khz

### CODE REVIEW

**Comment by leoscholl** [comment]
Changing the eye sampling rate highlights a problem with `get_kinematic_segements()`: how are we supposed to know the sampling rate of the data returned? does it matter?

File: aopy/preproc/oculomatic.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
+from .. import precondition
 from .. import data as aodata
 
-def parse_oculomatic(data_dir, files, debug=True):
+def parse_oculomatic(data_dir, files, samplerate=480, debug=True):
```
Comment (rcanfield2) [inline aopy/preproc/oculomatic.py:4]: Why choose an oculomatic sampling rate of 480? It might be useful to default to the sampling rate of other data

**Comment by rcanfield2** [comment]
> Changing the eye sampling rate highlights a problem with `get_kinematic_segements()`: how are we supposed to know the sampling rate of the data returned? does it matter?

We should probably have it output by the function.

**Review by rcanfield2 — APPROVED** [review]
Looks good! Sorry I forgot about this.

**Pull request merged**